.circleci/config.yml

```diff
@@ -1101,7 +1101,7 @@ jobs:
     # On large runs 2x faster than on medium. 3x on xlarge.
     <<: *base_ubuntu2004_xlarge
     environment:
-      <<: *base_ubuntu2204_xlarge_env
+      <<: *base_ubuntu2204_xlarge_env # TODO: Why is 2004 exending a 2204 env?
```

Comment:
Looks like a minor bug. Should have been `2204`. But the env is identical, which probably is why no one noticed.

Reply:
Figured as much, but wanted to double check. Thanks!